pep-0706.rst

```diff
@@ -564,10 +575,44 @@ Feature-wise, *tar format* and *UNIX-like filesystem* are essentially
 equivalent, so ``tar`` is a good name.
 
 
-Open Issues
-===========
+Possible Further Work
+=====================
 
-How far should this be backported?
+Adding filters to zipfile and shutil.unpack_archive
+---------------------------------------------------
+
+For consistency, :external+py3.11:mod:`zipfile` and
+:external+py3.11:func:`shutil.unpack_archive` could gain support
+for a ``filter`` argument.
+However, this would require research that this PEP's author can't promise
+for Python 3.12.
+
+Filters for ``zipfile`` would probably not help security.
+Zip is used primarily for cross-platform data bundles, and correspondingly,
+:external+py3.11:meth:`ZipFile.extract <zipfile.ZipFile.extract>`'s defaults
+are already similar to what a ``'data'`` filter would do.
+A ``'fully_trusted'`` filter, which would *newly allow* absolute paths and
+``..`` path components, might not be useful for much except
+a unified ``unpack_archive`` API.
+
+Filters should be useful for use cases other than security, but those
```

Comment:
```suggestion
Filters could be useful for use cases other than security, but those
```

Reply:
Thanks, but I do want *should* here -- you can lowercase/normalize names, strip an unwanted path component, etc.

Reply:
I wasn't sure, happy to leave this be as you prefer.